packages/samples/headless-react/src/components/search-parameter-manager/search-parameter-manager.class.tsx

```diff
@@ -0,0 +1,47 @@
+import {Component} from 'react';
```

Comment:
Does it need to be an example of a "React component" though ? 

Could it be an example, instead, on how you would use it in conjunction with something like react router ? 🤔 

Because it is pretty weird that everything is commented out :D 

If we can arrange for the router to be a `noop` behind the scene, even better. Nothing commented out, you have a full sample, and it does... "nothing" in the demo page.

Reply:
Good point, I'll research this

Reply:
I did some more research, and using React Router seems to add too much complexity, even requiring the `window.location` to be modified directly in order to avoid an infinitely recursive loop of `setState()` with the search parameter manager.

I tried using the search parameter manager with and without a React component and had the same issues using React Router either way.

In conclusion, I'll re-open this PR with the class and functional components removed in favor of moving that code directly inside `App.tsx`.